frontend/src/metabase/visualizations/echarts/cartesian/chart-measurements/index.ts

```diff
@@ -161,12 +235,41 @@ export const getChartPadding = (
   const yAxisNameTotalWidth =
     CHART_STYLE.axisName.size / 2 + CHART_STYLE.axisNameMargin;
 
-  if (chartModel.leftAxisModel?.label) {
-    padding.left += yAxisNameTotalWidth;
-  }
-  if (chartModel.rightAxisModel?.label) {
-    padding.right += yAxisNameTotalWidth;
-  }
+  const currentBoundaryWidth =
```

Comment:
For the boundary width calculation we still need to consider yAxisNameTotalWidth but you removed it above

Reply:
Fixed